pkg/compare/compare.go

```diff
@@ -91,39 +92,43 @@ func YAMLCmpWithIgnore(a, b string, ignorePaths []string) string {
 		return err.Error()
 	}
 
-	unmarshalConfigMapData(ao)
-	unmarshalConfigMapData(bo)
+	if kind, ok := ao["kind"]; ok && kind == "ConfigMap" {
```

Comment:
ok is redundant here and below. ao["kind"] will return "" if no present.

Reply:
Fixed, thanks!